code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -478,6 +478,10 @@
 			message = "<B>[src]</B> winks."
 			m_type = VISIBLE
 
+		if ("spin")
+			message = "<B>[src]</B> spins out of control!"

```

Comment:
Whoops, didn't notice that the tea reagent did it. Although the tea reagent really should be using SpinAnimation instead of that for loop.